docs/model/evidence/type.rst

```diff
@@ -21,16 +21,19 @@ Five Evidence Types are currently supported: Predictive (i.e. Therapeutic), Diag
      - Definition
    * - Predictive
      - |eye|
-     - Evidence pertaining to a variant’s effect on therapeutic response  
+     - Evidence pertaining to a variant's effect on therapeutic response  
    * - Diagnostic
      - |search|
-     - Evidence pertaining to a variant’s impact on patient diagnosis
+     - Evidence pertaining to a variant's impact on patient diagnosis
    * - Prognostic
      - |road|
-     - Evidence pertaining to a variant’s impact on disease progression, severity, or patient survival
+     - Evidence pertaining to a variant's impact on disease progression, severity, or patient survival
    * - Predisposing
      - |exclamation-triangle|
-     - Evidence pertains to a variant's role in conferring susceptibility to a disease
+     - Evidence pertains to a germline variant's role in conferring susceptibility to disease (including pathogenicity evaluations)
+   * - Oncogenic
+     - ||
```

Comment:
@jmcmichael - do we have an icon for Oncogenic that is available in civic-docs?

Reply:
yeah it's from the FontAwesome set that's used for the other icons, I'll see if I can add that real quick...